storage/s3/storage.go

```diff
@@ -18,3 +25,24 @@ func SetToContext(c *gin.Context, s3client s3iface.S3API) {
 func GetFromContext(c context.Context) s3iface.S3API {
 	return c.Value(key).(s3iface.S3API)
 }
+
+// PutS3Object uploads a object to s3.
+func PutS3Object(c context.Context, input *s3.PutObjectInput) (*s3.PutObjectOutput, error) {
+	return GetFromContext(c).PutObject(input)
+}
+
+// CreateTemplate uploads html file to s3.
+func CreateTemplate(c context.Context, tmplInput *entities.Template) error {
```

Comment:
Maybe UploadTemplate is better name for this function since is used in create nad update

Reply:
I think it should stay diff between is s3,storage as pkg? @gudgl